rust/store/src/lib.rs

```diff
@@ -228,6 +229,10 @@ impl StoreConnection {
         self.store.conn.read().unwrap().current_schema().to_edn_value()
     }
 
+    pub fn sync(&mut self, user_uuid: &Uuid) -> store_errors::Result<()> {
+        Ok(Syncer::flow(&mut self.handle, user_uuid).unwrap())
```

Comment:
Don't unwrap & repackage the result. Just return the result of `Syncer::flow` and do `map_err(|e| e.into())` if the result types don't match.